### PR TITLE
Merge the Nav Branch to the main in the NY Times Clone Project

### PR DESCRIPTION
**Pull request by MrkarlosM**
In this pull request we want to merge:

1. index.html file of the NY Times Clone:

- - Fixed navbar.

- - The main body.
- - Aside bar for related articles.
- - The footer part of related articles.
- - Footer part for links.

2. style.css file to stylish the HTML.
3. Modified README.md file, added name of the project, screenshot, live demo, and contact data of the owners of the project.
4. Modified LICENSE.md file, added as MIT license.

### CODE REVIEW

File: README.md
Comment (ElijahTobs) [inline README.md]: Correct the typo on line 25

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

### Hi @MrkarlosM and @AhmedAmin90 
### You guys have a very nice project and I must commend you for that :+1: 
### There's just a little more to do before your project can be approved and I'm sure once you make that change, it should be approved.

- [x] Some part of the Editor's Pick in the aside tag is out of the view port. Fix that and make sure there's no scroll bar at the bottom of the page.
- [x] Elements aligned with a combination of all three techniques ([float](https://css-tricks.com/all-about-floats/), [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and [grid](https://css-tricks.com/snippets/css/complete-guide-grid/)). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_pathwright.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/)) and ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_odin.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/positioning-and-floating-elements#introduction))
- [x] Check the inline comment below.

### Effect the changes and submit for another review.

## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes 🌳 ♲
Hi @MrkarlosM and @AhmedAmin90, 👏 You did very well on this project. Most requirements have done 👌, Kindly fix the following issues.
 [REQUIRED]

- [x] The NY times logo should be a link 

Once you fix the issues, kindly submit another code review ♲
 Happy coding 👨‍💻
 [Abdulaziz Redi Ali](https://azizali.ml/)

**Comment by AhmedAmin90** [comment]
I did the changed required point but I have a question.
I added a hover effect for nav buttons but Linters review said that I have to put : (.comment-btn button) before (nav button: hover) which does not make any sense ( because (.comment-btn button) is in another part of the page not related to nav section) so I removed it until asking why Linters said that?

**Review by mahmoud717 — APPROVED** [review]
## Project approved # 
**hi @MrkarlosM & @AhmedAmin90**
**good job implementing all of the required changes, you can go ahead and merge this pull request**
عاش يابوحميد

**Pull request merged**
